resources/js/frontend/wc-payplug-apple_pay-blocks.js

```diff
@@ -7,7 +7,7 @@ import {useEffect} from "react";
 import {apple_pay_update_payment, getPayment} from "./helper/wc-payplug-apple_pay-requests";
 import ApplePayCart from './wc-payplug-apple_pay_cart-blocks';
 
-const settings = getSetting( 'apple_pay_data', {} );
+let settings = getSetting( 'apple_pay_data', {} );
```

Comment:
why are you changing it to let if is not changed after?

Reply:
changed it back, it was a try for a solution to make it dynamic, forget to change it back